src/index.test.ts

```diff
@@ -68,30 +131,25 @@ describe('Geckoboard', () => {
     expect(async () => await gb.ping()).rejects.toThrow(
       new Error('Something went wrong with the request'),
     );
-    expect(() => mockAgent.assertNoPendingInterceptors()).not.toThrow();
   });
 
   it('can create a dataset', async () => {
-    const mockAgent = new MockAgent();
-    setGlobalDispatcher(mockAgent);
+    const dataset = prepareDatasetForCreation(mockPool);
```

Comment:
for the actual create test, I think it's better to see the input/output - but for the data tests using the helper makes sense to me 👍